.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
+  "env": {
+    "browser": true
+  },
+  "rules": {
+    "react/forbid-prop-types": 0,
```

Comment:
Should we try to have these enabled? Once you go "disable", you won't go "enable" ;)

Reply:
Sure thing!